src/esbmc/options.cpp

```diff
@@ -175,6 +176,13 @@ const struct group_opt_templ all_cmd_options[] = {
     {"bv", NULL, "use solver with bit-vector arithmetic"},
     {"ir", NULL, "use solver with integer/real arithmetic"},
     {"smtlib", NULL, "use SMT lib format"},
+    {"default-solver",
+     boost::program_options::value<std::string>()->value_name("<solver>"),
+     "override default solver used if no concrete one is specified"
```

Comment:
I don't get this option description. From the user's point of view, there is no difference between `--default-solver z3` and `--z3`, right?

Maybe `--solver z3`?

Reply:
This is rather a "meta-option" if you will. It adjusts the default/fallback solver in case no concrete solver (`--z3`) is given. I believe this is useful for running ESBMC with an unknown/diverse set of command line flags, which may or may not include specific solvers. Like our regression suite.
How could I improve the option description?

`--solver z3` would have a different meaning to my mind in that it does not modify our default fallback solver (Boolector, if built in).